microbench/src/main/java/io/netty/microbench/http2/internal/hpack/EncoderBenchmark.java

```diff
@@ -93,21 +111,10 @@ public void tearDown() {
 
     @Benchmark
     @BenchmarkMode(Mode.AverageTime)
-    public void encode(Blackhole bh) throws IOException {
-        Encoder encoder = new Encoder(maxTableSize);
+    public void encode(Blackhole bh) throws IOException, Http2Exception {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Refactor this method to throw at most one checked exception instead of: java.io.IOException, io.netty.handler.codec.http2.Http2Exception [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://garage.netty.io/sonarqube/coding_rules#rule_key=squid%3AS1160)